source/client/process_impl.cc

```diff
@@ -815,19 +822,25 @@ bool ProcessImpl::runInternal(OutputCollector& collector, const UriPtr& tracing_
     }
     tls_.registerThread(*dispatcher_, true);
     store_root_.initializeThreading(*dispatcher_, tls_);
+    absl::Status creation_status;
     runtime_loader_ = Envoy::Runtime::LoaderPtr{new Envoy::Runtime::LoaderImpl(
         *dispatcher_, tls_, {}, *local_info_, store_root_, generator_,
-        Envoy::ProtobufMessage::getStrictValidationVisitor(), *api_)};
-    ssl_context_manager_ =
-        std::make_unique<Envoy::Extensions::TransportSockets::Tls::ContextManagerImpl>(
-            time_system_);
+        Envoy::ProtobufMessage::getStrictValidationVisitor(), *api_, creation_status)};
+    if (!creation_status.ok()) {
+      ENVOY_LOG(error, "create runtime loader failed. Received bad status: {}",
+                creation_status.message());
+      return false;
+    }
 
     server_ = std::make_unique<NighthawkServerInstance>(
         admin_, *api_, *dispatcher_, access_log_manager_, envoy_options_, *runtime_loader_.get(),
         *singleton_manager_, tls_, *local_info_, validation_context_, grpc_context_,
         router_context_);
     server_factory_context_ =
         std::make_unique<NighthawkServerFactoryContext>(*server_, scope_root_);
+    ssl_context_manager_ =
```

Comment:
The `ssl_context_manager_` used to be initialized before the initialization of `server_`. This PR seems to be changing the order. Is the change necessary?

Reply:
Yes, I intentionally changed the order. `ssl_context_manager_` now requires `server_factory_context_` when instantiating. And `server_factory_context_` requires `server_` when instantiating.

Reply:
Thank you for explaining.